### ISSUE TITLE
`get_current_dir_name` not supported on OSX

### ISSUE THREAD

**Issue by rfbird** [kokkos/kokkos-tools#51]
`get_current_dir_name()` is not supported on osx. Would/should something like:

```
  #define CWD_MAX 1024                                                             
  char cwd[CWD_MAX];                                                               
  getcwd(cwd, CWD_MAX);                                                            
  printf("KokkosP: Kernel timing written to %s/%s \n", cwd, fileOutput);  
```

Be prefered for `kp_kernel_timer.cpp`? 

(As an aside, it would be nice if there was an option to write the file in plain text to avoid the need for `kp_reader` for an easier work flow on simple projects..)

**Comment by vlkale** [comment]
Possibly associated with #72

**Comment by dalg24** [comment]
Fixed in #57
